packages/WamExampleTemplate/src/index.js

```diff
@@ -31,7 +31,7 @@ export default class WamExampleTemplatePlugin extends WebAudioModule {
 
 	async createAudioNode(initialState) {
 		// DSP is implemented in WamExampleTemplateProcessor.
-		await WamExampleTemplateNode.addModules(this.audioContext, this._baseUrl);
+		await WamExampleTemplateNode.addModules(this.audioContext, this.groupId, this.moduleId);
```

Comment:
I think this is incorrect.  Pretty much everywhere else the method signature for `addModules` does not have `this.groupId`.

Reply:
You're right, thanks for catching that! It was left over from an earlier version and apparently I missed this one when I went through to update things. Looks like @Fr0stbyteR already fixed it